tdrs-frontend/src/actions/reports.js

```diff
@@ -14,11 +20,16 @@ export const upload = ({ file, section }) => async (dispatch) => {
       type: SET_FILE,
       payload: {
         fileName: file.name,
+        fileType: file.type,
         section,
+        uuid: uuidv4(),
       },
     })
   } catch (error) {
-    dispatch({ type: SET_FILE_ERROR, payload: { error, section } })
+    dispatch({
+      type: SET_FILE_ERROR,
+      payload: { error: Error({ message: 'something went wrong' }), section },
```

Comment:
Ensures a generic error is displayed to the user instead of the actual error in the `catch`